src/cargo/util/command_prelude.rs

```diff
@@ -378,6 +378,21 @@ pub trait CommandExt: Sized {
         )
         ._arg(unsupported_short_arg)
     }
+
+    fn arg_config(self) -> Self {
```

Comment:
Is it possible that we inline this function? It doesn't seem to be reused elsewhere.